graph-db/package.json

```diff
@@ -9,7 +9,8 @@
     "test:watch": "jest -i --watchAll",
     "test-coverage": "jest --coverage",
     "ci-test": "jest -i --reporters=\"default\" --reporters=\"jest-junit\"",
-    "bootstrap-local": "serverless invoke local -f bootstrapGremlinServer -e CMR_ROOT=\"https://cmr.uat.earthdata.nasa.gov\" -e PAGE_SIZE=2000 -e GREMLIN_URL=\"ws://localhost:8182/gremlin\" --stage=sit",
+    "bootstrap-local": "serverless invoke local -f bootstrapGremlinServer --path local-bootstrap.json -e CMR_ROOT=\"https://cmr.uat.earthdata.nasa.gov\" -e PAGE_SIZE=2000 -e GREMLIN_URL=\"ws://localhost:8182/gremlin\" --stage=sit",
+    "bootstrap-wss": "serverless invoke local -f bootstrapGremlinServer --path local-bootstrap.json -e TOKEN=\"replace_with_token_for_CMR_ROOT\" -e CMR_ROOT=\"https://cmr.uat.earthdata.nasa.gov\" -e PAGE_SIZE=20 -e GREMLIN_URL=\"wss://localhost:8182/gremlin\" --stage=sit",
```

Comment:
I don't like `replace_with_token_for_CMR_ROOT` being in this script like this, I can see it being too easy to accidentally commit a token. Instead use the readme file to show how to run the command and add the token in the command line, I think this would work (but double check the syntax works)
```
npm run bootstrap-wss -- -e TOKEN="replace_with_token_for_CMR_ROOT"
```

Reply:
Love this comment. Thanks!